packages/typespec-azure-resource-manager/test/rules/no-response-body.test.ts

```diff
@@ -33,10 +35,46 @@ describe("typespec-azure-resource-manager: no response body rule", () => {
       .toBeValid();
   });
 
+  it("is valid if 204 responses have no body", async () => {
+    await tester
+      .expect(
+        `
+        namespace Azure.MyService;
```

Comment:
Doesn't this cause the rule not to execute?

Reply:
no, this is what makes the rule to execute, this rule is only executed on Azure namespaces, this is also the core rule for reference: https://github.com/Azure/typespec-azure/blob/main/packages/typespec-azure-core/src/rules/no-response-body.ts

Reply:
Most ARM namespace start with 'Microsoft.`  Our namespaces in typespec-azure, like Azure.Core and Azure.ResourceManager start with `Azure.`